mmpose/models/heads/base_head.py

```diff
@@ -126,7 +126,13 @@ def decode(self, batch_outputs: Union[Tensor, Tuple[Tensor]],
             # Store the keypoint predictions in the data sample
             if 'pred_instances' not in data_sample:
                 pred_instances = InstanceData()
-                pred_instances.bboxes = data_sample.gt_instances.bboxes
+                if 'bboxes' in data_sample.gt_instances:
+                    pred_instances.bboxes = data_sample.gt_instances.bboxes
```

Comment:
When will the `bboxes` be absent, and can `bboxes` be replaced by `bbox_centers` and `bbox_scales` in the metrics, visualizer or other places where `bboxes` is used?

Reply:
Some face datasets, such as 300w, wflw, and aflw, do not provide `bbox`. In these cases the `bboxes` will be absent in `gt_instances`. 
Now we only compute NME on these datasets, where `bbox_centers` and `bbox_scales` are not used. They might be required if users want to plot predicted keypoints on original images. Do we need to support this function in visualizer?

Reply:
I suggest we add `bbox` in the data_info of these datasets, using `bbox_cs2xyxy`.